src/status_im2/contexts/profile/settings/constants.cljs

```diff
@@ -0,0 +1,64 @@
+(ns status-im2.contexts.profile.settings.constants
+  (:require [status-im2.common.not-implemented :as not-implemented]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn create-setting-item
+  [{:keys [title on-press icon]}]
+  (merge {:title    (i18n/label title)
+          :on-press on-press
+          :action   :arrow}
+         (when icon
+           {:image       :icon
+            :image-props icon})))
+
+(def settings-items
+  [[(create-setting-item {:title    :t/edit-profile
```

Comment:
whats the point of create-setting-item? why don't just have list of maps

Reply:
@flexsurfer they have some repeated fields like `:action` and `:icon` that i thought it's better to have a function to avoid repeat them

Reply:
but in this case you kinda hide it and make it more complex for other devs for later, it better to just have a plain data structure, even with duplicated keys

Reply:
renamed to list-items and removed the func, just used plain data structure